app/controllers/validators_controller.rb

```diff
@@ -69,7 +69,10 @@ def show
     # Grab the distances to show on the chart
     @vote_blocks = @val_histories.map(&:vote_distance).compact
 
-
+    @commission_histories = CommissionHistoryQuery.new(
```

Comment:
I think .count would be enough in this case. We're not displaying histories here, just checking if they exist.